### PR TITLE
[1.10.x] Refs #27924 -- Doc'd that cx_Oracle < 5.3 is required.

### PR DESCRIPTION
**Pull request by felixxm**
Ticket [27924](https://code.djangoproject.com/ticket/27924) (see [comment](https://code.djangoproject.com/ticket/27924#comment:3)).

### CODE REVIEW

File: docs/ref/databases.txt [hunk 1]
```diff
@@ -721,8 +721,8 @@ Oracle notes
 ============
 
 Django supports `Oracle Database Server`_ versions 11.2 and higher. Version
-4.3.1 or higher of the `cx_Oracle`_ Python driver is required, although we
-recommend version 5.1.3 or later as these versions support Python 3.
+between 4.3.1 and 5.2.1 of the `cx_Oracle`_ Python driver is required, although
```
Comment (timgraham) [inline docs/ref/databases.txt]: suggested wording: 
```
Versions 4.3.1 through 5.2.1 of the `cx_Oracle`_ Python driver are
supported, although 5.1.3 or later is recommended as these versions
support Python 3.
```

**Comment by felixxm** [comment]
@timgraham Thanks for review.

**Comment by timgraham** [comment]
Oh, and change `tests/requirements/oracle.txt` to `cx_oracle < 5.3`? Otherwise, please go ahead and merge to 1.10/1.9/1.8.

**Comment by felixxm** [comment]
You're right. I missed that.

File: tests/requirements/oracle.txt [hunk 2]
```diff
@@ -1 +1 @@
-cx_oracle
+cx_oracle<5.3
```
Comment (timgraham) [inline tests/requirements/oracle.txt]: Other requirements files are using spaces between the specifiers, e.g. `jinja2 >= 2.9.2`.

**Pull request merged**